alf/environments/suite_bsuite.py

```diff
@@ -0,0 +1,106 @@
+import alf
```

Comment:
Please add the copyright header at the top. See other files for an example.

Reply:
Added copyright header